packages/bpk-component-datepicker/src/BpkDatepicker.js

```diff
@@ -59,6 +73,9 @@ class BpkDatepicker extends Component {
     if (this.props.onDateSelect) {
       this.props.onDateSelect(dateObj);
     }
+    if (this.props.onClose) {
+      this.props.onClose();
+    }
```

Comment:
Likewise I think we could call `onClose` here instead of setting state and calling `props.onClose`